pkg/sass-parser/lib/src/expression/string.test.ts

```diff
@@ -284,5 +290,24 @@ describe('a string expression', () => {
     });
   });
 
-  it('toJSON', () => expect(utils.parseExpression('"foo"')).toMatchSnapshot());
+  describe('toJSON', () => {
+    it('a quoted string', () =>
+      expect(utils.parseExpression('"foo"')).toMatchSnapshot());
+
+    it('an unquoted string', () =>
+      expect(utils.parseExpression('foo')).toMatchSnapshot());
+
+    it('a string with interpolation', () =>
+      expect(utils.parseExpression('f#{o}o')).toMatchSnapshot());
+
+    it('supports()', () =>
+      expect(
+        (
+          (
+            scss.parse('@import "foo" supports(width: 1 + 1)')
+              .nodes[0] as ImportRule
+          ).imports.nodes[0] as StaticImport
+        ).modifiers!.nodes[1],
```

Comment:
I'm confused here. What is the value of `nodes[0]`, is it `supports(`?
And if so, is there a `nodes[2]` with the value of `)`?

Reply:
The first `nodes[0]` is the whole `@import` statement. The second one is `"foo" supports(width: 1 + 1)` (because `@import` in Sass can have multiple comma-separated clauses). Then `nodes[1]` is actually `supports(width: 1 + 1)`.